packages/reporter/src/commands/command.tsx

```diff
@@ -491,6 +491,10 @@ const Command: React.FC<CommandProps> = observer(({ model, aliasesWithDuplicates
     }
   }
 
+  if (model.name === 'request' && !appState.showFetchRequests) {
```

Comment:
I believe this will hide `cy.request` command logs as well (which I don't think we want). I think you just need to check `model.event` is `true`.